src/apiProxy/api-proxy.js

```diff
@@ -219,11 +248,16 @@ export const apiResponseDuotoneSetter = duotoneUrls => {
 	};
 };
 
+const MOCK_RESPONSE_OK = {
```

Comment:
this is _not_ a testing mock, but a mock `[http.IncomingMessage](https://nodejs.org/api/http.html#http_class_http_incomingmessage)` object to simulate the results of a real `externalRequest$`